OpenRA.Mods.Common/Traits/Turreted.cs

```diff
@@ -81,14 +81,17 @@ public virtual void Tick(Actor self)
 			else
 				realignTick = 0;
 
-			var df = DesiredFacing ?? (facing != null ? facing.Facing : TurretFacing);
-			TurretFacing = Util.TickFacing(TurretFacing, df, info.ROT);
+			self.World.AddFrameEndTask(w =>
```

Comment:
The only idea I have in mind is hackier, so yes, but no. Unity would do this behavior at the end of the tick too, so I really don't think it's that big of a deal.